### PR TITLE
Audible Hissing Sounds

### PR DESCRIPTION
**Pull request by Snakebittenn**
As on the tin.
Sounds from https://www.zapsplat.com/music/snake-hiss-short/ and https://www.zapsplat.com/music/snake-hissing/

### CODE REVIEW

**Comment by Snakebittenn** [comment]
!review

**Comment by alsoandanswer** [comment]
I thought we were against vocal emote sounds.

File: code/modules/emotes/definitions/audible.dm [hunk 1]
```diff
@@ -212,10 +212,22 @@
...
+	key = "hiss2"
+	emote_message_3p_target = "USER hisses at TARGET."
+	emote_message_3p = "USER hisses."
+	emote_sound = 'sound/voice/LizardHiss.ogg'
+
+/decl/emote/audible/lizhiss2
+	key = "hiss3"
+	emote_message_3p_target = "USER hisses at TARGET."
+	emote_message_3p = "USER hisses."
+	emote_sound = 'sound/voice/LizardHissShort.ogg'
```
Comment (NonQueueingMatt) [inline code/modules/emotes/definitions/audible.dm:229]: These should just be children of decl/emote/audible/hiss

**Comment by NonQueueingMatt** [comment]
> 
> 
> I thought we were against vocal emote sounds.

No? I don't think we ever were

**Comment by alsoandanswer** [comment]
> > I thought we were against vocal emote sounds.
> 
> No? I don't think we ever were

IIRC it was decided that sounds foe coughing and stuff were no bueno

**Pull request merged**